files/scripts/syncd.sh

```diff
@@ -150,6 +153,12 @@ stop() {
         TYPE=cold
     fi
 
+    if [[ x$sonic_asic_platform == x"mellanox" ]] && [[ x$TYPE == x"cold" ]]; then
+        debug "Stopping pmon service ahead of syncd..."
+        /bin/systemctl stop pmon
+        debug "Stopped pmon service."
```

Comment:
Remove trailing period to align with "Started pmon service" message

Reply:
fixed.